src/inject/dynamic-theme/stylesheet-modifier.ts

```diff
@@ -20,17 +20,22 @@ function getThemeKey(theme: Theme) {
     return themeCacheKeys.map((p) => `${p}:${theme[p]}`).join(';');
 }
 
-function getTempCSSStyleSheet(): {sheet: CSSStyleSheet; remove: () => void} {
+let tempStyle: HTMLStyleElement = null;
+
+function getTempCSSStyleSheet(): CSSStyleSheet {
+    if (tempStyle) {
+        return (tempStyle as HTMLStyleElement).sheet;
```

Comment:
Check if `tempStyle.parent === document.head`, if not, then it needs to be appended.

Reply:
Why is their a check needed? I've modified so it will only be appended to `document.head` As if you need a temp StyleSheet the DOM(head) is already loaded